classes/mail/SubmissionMailTemplate.inc.php

```diff
@@ -51,10 +51,10 @@ function assignParams($paramArray = array()) {
 		$request = Application::get()->getRequest();
 		parent::assignParams(array_merge(
 			array(
-				'submissionTitle' => strip_tags($submission->getLocalizedFullTitle()),
+				'submissionTitle' => htmlspecialchars($submission->getLocalizedFullTitle()),
```

Comment:
I think this will break places where some people write HTML into the submission title, and then customize their theme to show it correctly... Maybe it's better safe than sorry in emails, since an untrusted user could register and submit a dodgy title, but probably not get it published.

Reply:
I'd like to see us properly enforce expectations on titles until we officially support HTML there. When we add that support, we'll add a migration to explicitly escape existing content and anyone who has bodged their way through until now will have to edit and correct them one at a time (possibly in the database directly to avoid creating revisions). The less we facilitate 'works-for-now' the less pain we'll bake in for users who take the opportunity in the meantime.